tests/jballerina-unit-test/src/test/resources/test-src/object/client_resource_return_negative.bal

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Rename this file to `resource_method_return_type_negative.bal`

Reply:
Updated ebab2a74df0d4eddce0a69d7331556c045290bb9